src/components/EventItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dotdotdot from 'react-dotdotdot';
+import { capitalizeFirst } from 'utils';
+
+export const EventItem = ({
+  id,
+  name,
+  description,
+  eventDate,
+  timeRange,
+  place,
+}) => (
+  <div key={id} className="max-w-md mb-2">
+    <div className="px-6 py-4">
+      <div className="font-bold text-xl mb-1">
+        <a href={`https://www.facebook.com/events/${id}/`}>{name}</a>
+      </div>
+      <div className="text-l mb-2">
+        {capitalizeFirst(eventDate)}, {timeRange}, {place}
```

Comment:
You can use `.capitalize` css class instead (it's `text-transform: capitalize;`)